pkg/pb/gracefulexit.proto

```diff
@@ -15,35 +15,38 @@ service NodeGracefulExit {
   // GetSatellitesList returns a list of satellites that the storagenode has not begun a graceful exit for.
   rpc GetNonExitingSatellites(GetNonExitingSatellitesRequest) returns (GetNonExitingSatellitesResponse);
```

Comment:
nit - comment method name is incorrect
and maybe end with ...`that the storagenode has not exited.`